src/plugins/borrow-plugins/plugins/aave/AaveNetwork.ts

```diff
@@ -89,6 +89,7 @@ export const makeAaveNetworkFactory = (blueprint: AaveNetworkBlueprint): AaveNet
     //
 
     async getAssetPrice(address): Promise<BigNumber> {
+      // eslint-disable-next-line @typescript-eslint/return-await
```

Comment:
why the disable?

Reply:
The `otherMethods` are `any`, so ESLint doesn't see that the return value is a promise. We know the value is a promise, so we disable the rule.